juhpc

```diff
@@ -301,7 +303,8 @@ info "... done: wrapper created."
 info "Creating activate script..."
 
 export JULIAUP_DEPOT="$JULIAUP_INSTALLDIR/depot"
-export JULIA_DEPOT="$JULIAUP_INSTALLDIR/depot"
+# Added default $HOME/.julia path for backwards compatibility with exisitng Julia user installs
+export JULIA_DEPOT="$JULIAUP_INSTALLDIR/depot:$HOME/.julia"
```

Comment:
I'm not convinced that it is a good idea to mix with installations. Also, we can have many different JUHPC setups that don't interfere with each other; adding this common depot or would break this great feature.